packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/helpers/golden_helper.dart

```diff
@@ -17,69 +23,56 @@ const localizations = [
   ...GlobalMaterialLocalizations.delegates,
 ];
 
-/// return a [LabeledDeviceBuilder] with a scenario rendered on all device sizes
-///
-/// [widget] - to be rendered in the golden master
-///
-/// [scenario] - [Scenario] which will be added to [DeviceBuilder]
-LabeledDeviceBuilder generateDeviceBuilder({
+const _defaultDevices = [
+  Device.phone,
```

Comment:
@RomanovaPrime, please suggest a default list here

Reply:
@StanevPrime Values are taken from previous implementation, but always welcome to change them.

Reply:
Here is a set of devices, that we use is ProCredit application for some of tests.
final devices = [
    const Device(
        name: 'iPhone SE(2nd generation)',
        size: Size(375, 667),
        safeArea: EdgeInsets.only(top: 20),
        devicePixelRatio: 2),
    const Device(name: 'Google Pixel 4a', size: Size(412, 732)),
    const Device(
        name: 'iPhone 13 mini',
        size: Size(375, 812),
        safeArea: EdgeInsets.only(top: 44, bottom: 34),
        devicePixelRatio: 3),
    const Device(
        name: 'Google Pixel 5',
        size: Size(393, 851),
        safeArea: EdgeInsets.only(top: 24, bottom: 48),
        devicePixelRatio: 2.75),
    const Device(
        name: 'Samsung Galaxy S20',
        size: Size(412, 915),
        safeArea: EdgeInsets.only(top: 24, bottom: 48),
        devicePixelRatio: 3),
    const Device(
        name: 'Apple iPad Pro 12.9',
        size: Size(1024, 1366),
        safeArea: EdgeInsets.only(top: 24, bottom: 34),
        devicePixelRatio: 2),
    const Device(
        name: 'Samsung Galaxy Tab S6 Landscape',
        size: Size(1280, 800),
        safeArea: EdgeInsets.only(top: 24, bottom: 48),
        devicePixelRatio: 2),
  ];